### DISCUSSION TITLE
REWORKD_PLATFORM_REPLICATE_API_KEY="changeme"

### DISCUSSION THREAD

**Discussion by yhyu13** [reworkd/AgentGPT#760] [Q&A]
Maybe a stupid question, what should fill in the replicate API key? What does it do? Where can I obtain one replicate API key?

THanks!

**Comment by kanrun** [comment]
me too dont know

**Comment by tamaosaka** [comment]
Is this the place to ask me questions?
Sorry if not.

**Comment by zimmra** [comment]
Model repo to run various AI models, like image generation, etc. 

https://replicate.com/ 
https://replicate.com/account/api-tokens
